src/Stryker.CLI/Stryker.CLI/ConfigHandler.cs

```diff
@@ -0,0 +1,126 @@
+            inputs.ConcurrencyInput.SuppliedInput = config.Concurrency;
+
+            inputs.SinceInput.SuppliedInput =
+                config.Since is not null &&
+                (config.Since.Enabled.HasValue && config.Since.Enabled.Value);
+
+            inputs.WithBaselineInput.SuppliedInput =
+                config.Baseline is not null &&
+                (config.Baseline.Enabled.HasValue && config.Baseline.Enabled.Value);
+
+            inputs.BaselineProviderInput.SuppliedInput = config.Baseline?.Provider;
+            inputs.DiffIgnoreChangesInput.SuppliedInput = config.Since?.IgnoreChangesIn;
+            inputs.FallbackVersionInput.SuppliedInput = config.Baseline?.FallbackVersion;
+            inputs.AzureFileStorageUrlInput.SuppliedInput = config.Baseline?.AzureFileShareUrl;
+            inputs.CoverageAnalysisInput.SuppliedInput = config.CoverageAnalysis;
+            inputs.DisableBailInput.SuppliedInput = config.DisableBail;
+            inputs.DisableMixMutantsInput.SuppliedInput = config.DisableMixMutants;
+            inputs.AdditionalTimeoutInput.SuppliedInput = config.AdditionalTimeout;
+            inputs.MutateInput.SuppliedInput = config.Mutate;
+            inputs.MutationLevelInput.SuppliedInput = config.MutationLevel;
+            inputs.ProjectNameInput.SuppliedInput = config.ProjectInfo?.Name;
+            inputs.ModuleNameInput.SuppliedInput = config.ProjectInfo?.Module;
+            inputs.ProjectVersionInput.SuppliedInput = config.ProjectInfo?.Version;
+            inputs.ReportersInput.SuppliedInput = config.Reporters;
+
+            inputs.SinceTargetInput.SuppliedInput = config.Since?.Target;
+            inputs.SolutionInput.SuppliedInput = config.Solution;
+            inputs.TargetFrameworkInput.SuppliedInput = config.TargetFramework;
+
+            inputs.ProjectUnderTestNameInput.SuppliedInput = config.Project;
+            inputs.ThresholdBreakInput.SuppliedInput = config.Thresholds?.Break;
+            inputs.ThresholdHighInput.SuppliedInput = config.Thresholds?.High;
+            inputs.ThresholdLowInput.SuppliedInput = config.Thresholds?.Low;
+            inputs.VerbosityInput.SuppliedInput = config.Verbosity;
+            inputs.LanguageVersionInput.SuppliedInput = config.LanguageVersion;
+            inputs.TestProjectsInput.SuppliedInput = config.TestProjects;
+            inputs.TestCaseFilterInput.SuppliedInput = config.TestCaseFilter;
+            inputs.DashboardUrlInput.SuppliedInput = config.DashboardUrl;
+            inputs.IgnoreMutationsInput.SuppliedInput = config.IgnoreMutations;
+            inputs.IgnoredMethodsInput.SuppliedInput = config.IgnoreMethods;
+
+            inputs.ReportFileNameInput.SuppliedInput = config.ReportFileName;
+        }
+
+        private static FileBasedInput LoadConfig(string configFilePath)
+        {
+            var lines = File.ReadAllLines(configFilePath);
+            // filter json comments
```

Comment:
So I'm confused a bit: here you write that we can't handle trailing comments but then you add a trailing comment in the test json config file - and things work?

Reply:
I have solved this by reverting to separate deserializers.